packages/reporter/cypress/integration/header_spec.ts

```diff
@@ -31,30 +31,6 @@ describe('header', () => {
     })
   })
 
-  describe('tests button', () => {
```

Comment:
Ideally, these tests should move somewhere, but I couldn't find any tests that display both the Vue & React portions of this. When only the React parts get displayed (like in these tests), the button doesn't get displayed at all. If anybody has an idea for where to put them, i can move/rewrite them.

Reply:
I was thinking about this this morning actually, I think our new e2e tests for the app should be able to show this, will double check.

Reply:
confirmed we don't have the ability to do this in e2e test atm, at least not easily. we can add a vue CT test though that renders the portal and makes sure the Vue part works correctly.

Reply:
We need to work on supporting E2E testing for that page...